cibyl/outputs/cli/ci/system/common/builds.py

```diff
@@ -94,3 +94,30 @@ def get_duration_section(palette, build):
     text[-1].append(f'{as_minutes(build.duration.value):.4f}min')
 
     return text.build()
+
+
+def print_stage(stage, palette, verbosity=0):
+    """
+        Generate string representation of a Stage model.
+
+        :param test: The stage.
+        :type test: :class:`cibyl.models.ci.base.stage.Stage`
+        :param palette: The palette of colors to follow.
+        :type palette: :class:`cibyl.utils.colors.ColorPalette`
+        :param verbosity: The verbosity level to use.
+        :type verbosity: int
+        :return: Textual representation of the provided model.
+        :rtype: str
+    """
+    printer = IndentedTextBuilder()
+
+    printer.add(palette.blue('- '), 0)
+    printer[-1].append(stage.name.value)
+
+    if verbosity > 0:
+        if stage.status.value:
+            printer.add(get_status_section(palette, stage), 1)
+
+        if stage.duration.value:
+            printer.add(get_duration_section(palette, stage), 1)
```

Comment:
I am not so sure of having this function here. This module is meant for storing common functions meant for printing builds, not stages.

Also, using these two functions can be troublesome down the line. You are passing a type is not meant to be. It works because they have the same attributes, but I don't know.